### ISSUE TITLE
Transport packages should not be signed

### ISSUE THREAD

**Issue by krwq** [dotnet/corefx#39603]
Per discussion here https://github.com/dotnet/corefx/pull/36022#discussion_r265347843
we should not sign symbol and *.Private.* packages as it adds some observable amount of time to the overall signing time (few minutes?) - we currently do it in parallel with testing so overall time might be similar.

Related code:
https://github.com/dotnet/corefx/blob/master/eng/Signing.props#L13

cc: @mmitche

**Comment by mmitche** [comment]
Thanks

**Comment by blowdart** [comment]
Are these files distributed? Or are they thrown away at some point?

**Comment by blowdart** [comment]
Also @GrabYourPitchforks

**Comment by mmitche** [comment]
@blowdart Those aren't, they are pulled apart and their internals are repackaged. They never make it to nuget.org,  So the binaries inside should be signed, but we don't need to sign the packages themselves.

**Comment by blowdart** [comment]
Ah so the innards remain signed? This is just something to shuttle things arounds?

**Comment by safern** [comment]
> Are these files distributed? Or are they thrown away at some point?

They are consumed by core-setup... also, most of the contents in those packages, are crossgen'd by core-setup before re-packaging and are signed again in core-setup, so we're double signing some pieces. (I believe the runtime assemblies are the one crossgen'd, but not the ref assemblies).

cc: @dagood @ericstj

**Comment by blowdart** [comment]
Ok I shall wind down then :)

**Comment by mmitche** [comment]
Right. The goal is just to get a minimal set of signed files, because signing is a big part of the build right now (adds about an hour to the build time, taking more time than it takes to actually build the code)

**Comment by blowdart** [comment]
Well of course signing is slow, that's how you know it's secure ;)

**Comment by mmitche** [comment]
> Well of course signing is slow, that's how you know it's secure ;)
@blowdart Oh, that makes so much sense.  Now I know how to implement certificate verification.

```
bool IsValidCert(X509Certificate2 cert) {
    Thread.Sleep(5000);
    return true;
}
```
